src/components/ui/ListEntry.js

```diff
@@ -64,7 +65,7 @@ ListEntry.displayName = 'ListEntry'
 
 ListEntry.propTypes = {
   leftIcons: PropTypes.node,
-  primaryText: PropTypes.string,
+  primaryText: PropTypes.node,
```

Comment:
Thanks!! Maybe we should redesign this so it's not just text (either take *children* or (primaryText and/or secondaryText)), but we can do that in the future.

Reply:
Okay, so keep it as string, then right?

Reply:
Node is good! I just mean we can make further improvements in a future PR.